src/torrent/data/file_list.cc

```diff
@@ -592,7 +592,7 @@ FileList::create_chunk(uint64_t offset, uint32_t length, bool hashing, int prot)
   if (offset + length > m_torrent_size)
     throw internal_error("Tried to access chunk out of range in FileList", data()->hash());
 
-  auto chunk = std::make_unique<Chunk>();
+  auto chunk = new Chunk;
```

Comment:
This isn't what I meant earlier, this is a use case that make_unique is meant for.

Reply:
makes no sense but OK.

Reply:
It makes no sense because you do not understand my point about why make_unique is superfluous for `m_client_list(std::make_unique<ClientList>())`.

Reply:
I get it for the constructor. Not so much if creating a unique_ptr and releasing it at the end of a function.